.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -20,11 +20,11 @@ If this is a backport PR (PR made against branches other than `main`),
 please ensure that the PR title is in the following format:
 
 ```
-[X.Y] <title from the original PR> (GH-NNNN)
+[X.Y] <title from the original PR> (GH-NNNNN)
```

Comment:
PR numbers are currently six digits long :)

Reply:
I... think I don't know how to count anymore :')